apluslms_roman/configuration.py

```diff
@@ -1,19 +1,27 @@
+from itertools import chain
 from os import listdir
-from os.path import join, isdir, isfile
+from os.path import basename, join, isdir, isfile, relpath
 
 from apluslms_yamlidator.document import Document
 from apluslms_yamlidator.utils.collections import OrderedDict
 from apluslms_yamlidator.utils.version import Version
 
 from .utils.translation import _
 
-DEFAULT_NAME = 'course'
+DEFAULT_NAME = 'roman'
```

Comment:
This is not used within the file.

Reply:
it was used in cli but I suppose it isn't needed if we have default_filename